deploy/kubedirector/kubedirector.hpe.com_kubedirectorapps_crd.yaml

```diff
@@ -136,7 +136,7 @@ spec:
                     type: array
                     items:
                       type: string
-                      pattern: '^configure$|^addnodes$|^delnodes$'
+                      pattern: '^configure$|^addnodes$|^delnodes$|^upgrade$'
```

Comment:
Not related to your change, but I see a bug here where this list should also include the reconnect event. Let's not fix it here though, I will file a separate JIRA to not only fix the spec but also examine the reconnect event code to make sure it honors the eventList spec.

Reply:
Discarded this change